From 0ac45feb2a9144173c976e7f1f4d99ffac0760f5 Mon Sep 17 00:00:00 2001
From: Jeff Moyer <jmoyer@redhat.com>
Date: Thu, 28 Aug 2014 19:15:51 -0400
Subject: [virt] virtio_blk: Add missing 'static' qualifiers

Message-id: <1409253371-23161-2-git-send-email-jmoyer@redhat.com>
Patchwork-id: 89847
O-Subject: [RHEL7 PATCH 01/21] virtio_blk: Add missing 'static' qualifiers
Bugzilla: 1105698
RH-Acked-by: Fam Zheng <famz@redhat.com>

This is a backport of the following upstream commit.
Related to bug 1105698.

  commit 2a647bfe1bf43af51530862b9ec70032bdd0a44c
  Author: Jonghwan Choi <jhbird.choi@samsung.com>
  Date:   Mon May 20 10:25:39 2013 +0930

    virtio_blk: Add missing 'static' qualifiers

    Add missing 'static' qualifiers

    Signed-off-by: Jonghwan Choi <jhbird.choi@samsung.com>
    Acked-by: Michael S. Tsirkin <mst@redhat.com>
    Signed-off-by: Rusty Russell <rusty@rustcorp.com.au>

Signed-off-by: Jeff Moyer <jmoyer@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/block/virtio_blk.c b/drivers/block/virtio_blk.c
index f3cb1a2..189c286 100644
--- a/drivers/block/virtio_blk.c
+++ b/drivers/block/virtio_blk.c
@@ -20,7 +20,7 @@ module_param(use_bio, bool, S_IRUGO);
 static int major;
 static DEFINE_IDA(vd_index_ida);
 
-struct workqueue_struct *virtblk_wq;
+static struct workqueue_struct *virtblk_wq;
 
 struct virtio_blk
 {
-- 
1.7.1